### DISCUSSION TITLE
Yo, can I help you with this?

### DISCUSSION THREAD

**Discussion by OOFMAN29803** [kurisubrooks/tfnsw-pids#4] [General]
I love transport and would love to help

**Comment by kurisubrooks** [comment]
Sure, I'm happy to accept pull requests. I was considering a Next.js rewrite to solve the CORS issues but I can't devote the time right now. What sort of things would you consider changing?

**Reply by OOFMAN29803** [reply]
Well since it was transport based im thinking of working on a Sydney ferries PID but I am also trying to work through CORS, i am probably gonna work on websites if you care for that, but um yea I just want to help expand
